lib/types/CredentialIssuance.types.ts

```diff
@@ -57,7 +55,30 @@ export interface JWK {
   n?: string;
 }
 
-export type Alg = 'ES256' | 'EdDSA';
+export interface ProofOfPossessionCallbackArgs {
+  privateKey: unknown;
```

Comment:
Public and private key should not be exposed at all. It is up to the implementation how to handle that. I would expect a kid/did in there though

Reply:
I've removed this because we have `[x: string]: unknown;` listed.